cinema/views.py

```diff
@@ -43,11 +52,45 @@ def get_serializer_class(self):
 
         return MovieSerializer
 
+    def get_queryset(self):
+        queryset = Movie.objects.prefetch_related("actors", "genres")
+        actor_ids = self._turn_params_to_int(
+            self.request.query_params.getlist("actors")
```

Comment:
just use `get`, since params will be provided as `key - value` pair, and value can be list of strings. Using get you will extract it

Reply:
> just use `get`, since params will be provided as `key - value` pair, and value can be list of strings. Using get you will extract it

I will fix. But honestly I can't see big difference in implementation. When I use `getlist` I have list instantly and do not have to use `split` method in `_turn_params_to_int`